documentation/dataInsertion/pythonModule/quickStart.md

```diff
@@ -69,18 +36,18 @@ my_debiai = debiai.Debiai(DEBIAI_BACKEND_URL)
 debiai_project = my_debiai.create_project(DEBIAI_PROJECT_NAME)
 ```
 
-A project named "Hello DebiAI" is now created. To create an other project, change DEBIAI_PROJECT_NAME.
```

Comment:
Maybe obvious, but still... ;)

Reply:
I have added it back, take a look at the last commit